src/services/getPopularTagLists.ts

```diff
@@ -0,0 +1,32 @@
+import { countArticleTags } from "@/utils/countArticleTags";
+
+export const getPopularTagLists = async () => {
+  try {
+    const endPoint = process.env.NEXT_PUBLIC_GET_ALL_ARTICLES_END_POINT;
+
+    if (!endPoint) {
+      throw new Error("엔드포인트가 존재하지 않습니다!");
+    }
+
+    const res = await fetch(endPoint);
+
+    if (!res.ok) {
+      throw new Error("응답이 올바르지 않습니다!");
+    }
+
+    const data = await res.json();
+
+    const { articles } = data;
+
+    const popularTags = countArticleTags({ articles });
+
+    if (!popularTags) {
+      return [];
+    } else {
+      return popularTags;
+    }
```

Comment:
```suggestion
    const popularTags = countArticleTags({ articles }) || [];
    return popularTags;
```

요건 어떨까요?
그리고 상단의 코드 라인들이 다 한 칸씩 떨어져 있는데 조건문이냐, 같은 관심사냐, 그루핑된 맥락이냐에 따라 합쳐도 좋을 것 같습니다용